library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -93,6 +94,29 @@ protected function assembleTitle(BaseHtmlElement $title): void
             $this->createSubject(),
             Html::tag('span', ['class' => 'state-text'], $this->state->getStateTextTranslated())
         ));
+
+        if ($this->state->affects_children && $this->item->affected_children) {
+            $total = $this->item->affected_children;
+
+            if ((int) $total > 1000) {
+                $total = '1000+';
+            }
+
+            $icon = new Icon(Icons::HOST_DOWN);
```

Comment:
I'd rather want to see an icon for the unreachable state, which just happens to be equal to the host's icon.